src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for printable strings
+ * 
+ */
+
```

Comment:
avoid leaving a line of space between header comments and the interface/method/class